
R version 4.3.0 (2023-04-21) -- "Already Tomorrow"
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: x86_64-pc-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type 'license()' or 'licence()' for distribution details.

R is a collaborative project with many contributors.
Type 'contributors()' for more information and
'citation()' on how to cite R or R packages in publications.

Type 'demo()' for some demos, 'help()' for on-line help, or
'help.start()' for an HTML browser interface to help.
Type 'q()' to quit R.

> library(testthat)
> library(roxygen2)
> 
> test_check("roxygen2")
[WARNING] Duplicate link reference '[roxygenize()]' at file16c8b8d9ffd9.knit.md line 8 column 1
[WARNING] Duplicate link reference '[stringr::str_length()]' at file16c8b8d9ffd9.knit.md line 9 column 1
[ FAIL 2 | WARN 540 | SKIP 87 | PASS 772 ]

══ Skipped tests ═══════════════════════════════════════════════════════════════
• On CRAN (87)

══ Failed tests ════════════════════════════════════════════════════════════════
── Error ('test-object-from-call.R:171:3'): can define constructor with R.oo ───
<packageNotFoundError/error/condition>
Error in `loadNamespace(x)`: there is no package called 'R.oo'
Backtrace:
    ▆
 1. ├─roxygen2:::call_to_object(...) at test-object-from-call.R:171:2
 2. │ └─base::eval(code, envir = env)
 3. │   └─base::eval(code, envir = env)
 4. └─base::loadNamespace(x) at test-object-from-call.R:172:4
 5.   └─base::withRestarts(stop(cond), retry_loadNamespace = function() NULL)
 6.     └─base (local) withOneRestart(expr, restarts[[1L]])
 7.       └─base (local) doWithOneRestart(return(expr), restart)
── Error ('test-object-from-call.R:179:3'): can define method for R.methodsS3 ──
<packageNotFoundError/error/condition>
Error in `loadNamespace(x)`: there is no package called 'R.methodsS3'
Backtrace:
    ▆
 1. ├─roxygen2:::call_to_object(...) at test-object-from-call.R:179:2
 2. │ └─base::eval(code, envir = env)
 3. │   └─base::eval(code, envir = env)
 4. └─base::loadNamespace(x) at test-object-from-call.R:180:4
 5.   └─base::withRestarts(stop(cond), retry_loadNamespace = function() NULL)
 6.     └─base (local) withOneRestart(expr, restarts[[1L]])
 7.       └─base (local) doWithOneRestart(return(expr), restart)

[ FAIL 2 | WARN 540 | SKIP 87 | PASS 772 ]
Error: Test failures
Execution halted
